sql/item_mods.sql

```diff
@@ -67293,6 +67293,13 @@ INSERT INTO `item_mods` VALUES (27594,288,4); -- DOUBLE_ATTACK: 4
 INSERT INTO `item_mods` VALUES (27595,1,12);  -- DEF: 12
 INSERT INTO `item_mods` VALUES (27595,28,12); -- MATT: 12
 
+-- Emico Mantle
+INSERT INTO `item_mods` VALUES (27597,1,12);     -- DEF: 12
+INSERT INTO `item_mods` VALUES (27597,2,20);     -- HP: 20
+INSERT INTO `item_mods` VALUES (27597,10,3);     -- VIT: 3
+INSERT INTO `item_mods` VALUES (27597,18,10);    -- EARTH_RES: 10
+INSERT INTO `item_mods` VALUES (27597,27,4);     -- ENMITY: 4
+
 -- Dew Silk Cape
```

Comment:
You did not create a different branch for the sale values? This is already in your other PR.

Reply:
Did I need to because it was of the same thing. Just a minor -- note update.

Reply:
> I thought I did. Carver was walking me through and we did make a branch of it.
> 
> edit: wait, why is this edit here too? Eh, i'm still learning this structuring of this. My apologies. I may have likely not a proper pull before starting the new branch therefore catching the other edit. Again, my apologies. Rough start. Lol

yeah, remember, last thing i told you last night is to make sure you switch back to your base each time, as this is what will happen if you dont :) 

I can help you fix this later on...